src/index.web.ts

```diff
@@ -0,0 +1,72 @@
+export const MMKV = {
+  /**
+   * Set a value for the given `key`.
+   */
+  set: (key: string, value: boolean | string | number): void => {
+    localStorage.setItem(key, value.toString());
+  },
+  /**
+   * Get a boolean value for the given `key`.
+   *
+   * @default false
+   */
+  getBoolean: (key: string): boolean => {
+    var result = localStorage.getItem(key);
+
+    if (result) {
+      return Boolean(result);
+    }
+
+    return false;
+  },
+  /**
+   * Get a string value for the given `key`.
+   *
+   * @default undefined
+   */
+  getString: (key: string): string | undefined => {
+    var result = localStorage.getItem(key);
+
+    if (result) {
+      return result;
+    }
+
+    return undefined;
```

Comment:
I feel like this can simply be 
```js
return result;
```

Reply:
Unfortunately, since local storage returns null instead of undefined if there is no result, I have to do to this to maintain the same functionality.

Reply:
How about:

```suggestion
    return result ?? undefined;
```

Reply:
Actually the check above should also have been changed to 

```typescript
if (result != null) {
```

Otherwise empty string will fall into the undefined / null path.